pipeline/transform/compute_metrics.py

```diff
@@ -14,4 +14,19 @@ def compute_internal_matching_metrics(
     metric_output = metric.count_proposed_matches_per_user(
         output.users, iter(output.internal_data.ranked_matches)
     )
-    return MatchingMetrics(n_proposed_matches_per_user=metric_output)
+    metric_match_distribution = metric.intent_distribution_in_selected_matches(
+        output.matches
+    )
+    (
+        metric_community_distribution_1,
```

Comment:
Use more informative variable names here. This should describe what this variable is intended to store -- the seeking and giving intent community distribution.